apps/hodu-prod/hodu/hodu-server.yaml

```
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hodu-server
+  labels:
+    app: hodu-server
+  namespace: hodu-prod
+spec:
+  replicas: 0
+   selector:
+    matchLabels:
+      app: hodu-server
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+  revisionHistoryLimit: 4
+  template:
+    metadata:
+      labels:
+        app: hodu-server
+    spec:
+      serviceAccountName: hodu-server
+      containers:
```

isolate 실행하기 위한 최소 조건입니다. privileged 옵션 추가해주세요.
```suggestion
      containers:
        securityContext:
          privileged: true
```

---

@wafflestudio/waffle-k8s-admin 이거 추가해도 괜찮을까요??
파드 내에 샌드박스 환경을 구축하기 위해 필요합니다

---

@Hank-Choi 에구 이 댓글을 지금 봤네요 ㅠㅠ
넵 퍼블릭한 서버가 맞고, cgroup 을 사용하기 위한 최소한의 조건이라 다른 방법이 없습니다.
1. isolate → cgroup 에 의존,
2. cgropu → privileged 필요입니다.

관련 링크 첨부합니다.
- https://github.com/kubernetes/kubernetes/issues/121190 (k8s 에서 privileged 없이 cgroup 사용이 가능하냐는 질문)
- https://github.com/ioi/isolate/issues/35#issuecomment-335031399 (isolate 를 구동하기 위해선 반드시 privileged 옵션이 필요하다는 글)